openslides_backend/action/actions/user/update.py

```diff
@@ -42,5 +42,6 @@ class UserUpdate(
             "vote_delegated_$_to_id",
             "vote_delegations_$_from_ids",
             "group_$_ids",
+            "is_demo_user",
```

Comment:
I see you are also doing https://github.com/OpenSlides/openslides-backend/issues/722 in this PR ;) please assign yourself there and link this issue, I assume you also included it in the permission check then

Reply:
saw that you did, I will go ahead and assign you then.